sigle/src/modules/layout/components/DashboardLayout.tsx

```diff
@@ -1,257 +1,182 @@
-            <StyledLink href={sigleConfig.discordUrl} target="_blank">
-              <DiscordLogoIcon
-                height={iconSize}
-                width={iconSize}
-                className="icon"
-              />
-            </StyledLink>
-            <StyledLink href={sigleConfig.githubUrl} target="_blank">
-              <GitHubLogoIcon
-                height={iconSize}
-                width={iconSize}
-                className="icon"
-              />
-            </StyledLink>
-          </MenuBottomItemSocial>
-        </MenuBottom>
-      </div>
-    </SidebarContent>
-  );
-};
-
-interface Props {
-  children: React.ReactNode;
-}
+  let triggerName;
+
+  switch (router.pathname) {
+    case '/':
+      triggerName = 'Drafts';
+      break;
+    case '/published':
+      triggerName = 'Published';
+      break;
+    case '/settings':
+      triggerName = 'Settings';
+      break;
+    default:
+      triggerName = 'Drafts';
+      break;
+  }
 
-export const DashboardLayout = ({ children }: Props) => {
   return (
-    <Container>
-      <Sidebar>
-        <DashboardSidebar />
-      </Sidebar>
-      <ContentContainer>
-        <Content>{children}</Content>
-      </ContentContainer>
-    </Container>
+    <FullScreen>
+      <AppHeader />
+      <Container
+        css={{
+          display: 'grid',
+          flex: 1,
+          mt: '$10',
+          width: '100%',
+
+          '@md': {
+            justifyContent: 'center',
+            gridTemplateColumns: '724px',
+          },
+
+          '@lg': {
+            gridTemplateColumns: '834px',
+          },
+
+          '@xl': {
+            gridTemplateColumns: '1fr 826px 1fr',
+          },
+        }}
+      >
+        <Sidebar
+          css={{
+            display: 'none',
+            '@xl': {
+              display: 'flex',
+            },
+          }}
+        >
+          <Link href="/" passHref>
+            <NavItem selected={router.pathname === '/'}>Drafts</NavItem>
+          </Link>
+          <Link href="/published" passHref>
+            <NavItem selected={router.pathname === '/published'}>
+              Published
+            </NavItem>
+          </Link>
+          <Link href="/settings" passHref>
+            <NavItem selected={router.pathname === '/settings'}>
+              Settings
+            </NavItem>
+          </Link>
+        </Sidebar>
+        <Box
+          css={{
+            mb: '$5',
+            mx: 'auto',
+            width: '100%',
+          }}
+        >
+          <Accordion
+            css={{ '@xl': { display: 'none' } }}
+            collapsible
+            type="single"
+          >
+            <AccordionItem value="item1">
+              <AccordionTrigger>{triggerName}</AccordionTrigger>
+              <AccordionContent>
+                <Link href="/" passHref>
+                  <NavItem hidden={router.pathname === '/'} variant="accordion">
+                    Drafts
+                  </NavItem>
```

Comment:
If the goal is to hide the element would doing smth like this in react make sense?
```
                {router.pathname !== '/' ? <Link href="/" passHref>
                  <NavItem variant="accordion">
                    Drafts
                  </NavItem> : null}
```

Reply:
Ah yep that makes more sense 😅